_includes/manuals/2.1/3.5.1_initial_setup.md

```diff
@@ -22,21 +22,6 @@ foreman-rake db:seed</pre>
 For more information please see the database configuration page
 [here](manuals/{{page.version}}/index.html#3.5.3DatabaseSetup)
 
-#### Import Data from Puppet
-
-At this point, you might want to go through the [[FAQ]] to see how can you import your data into Foreman.
-
-#### Start The Web Server
-
-if you installed via rpm, just start the foreman service, or start the builtin web server by typing:
-<pre>RAILS_ENV=production rails server</pre>
```

Comment:
For what it's worth, `systemctl start foreman` would be the modern equivalent if you installed the `foreman-service` package.